meta-lmp-base/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -1,5 +1,42 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
+# Based on the original recipe but changed for LmP (done to avoid changing via removal)
```

Comment:
I'd add there a reminder for this variable having to be maintained and updated from the main recipe.

Reply:
Done.